docs/3.0.0-beta.x/getting-started/install-requirements.md

```diff
@@ -1,127 +1,27 @@
-## You should see "v10.x.x
-```
-
-Next, verify npm has correctly installed:
-
-```shell
-npm -v
-## You should see "6.x.x"
-```
-
-:::
-
-::: tab "Mac O/S"
-
-### Installing Node.js on Mac O/S X (Mojave)
-
-There are multiple methods to install Node.js on _Mac O/S X (Mojave)_.
-
-We will follow the most common download and installation procedure. _These instructions are for Mac O/S X (Mojave)_. (If you are installing on a different version of Mac O/S or if you have trouble following these instructions, please review the [official Node.js documentation](https://nodejs.org/en/docs/).)
+You can also use **yarn** if you want [here](https://yarnpkg.com/en/docs/getting-started) are the instructions to get started with it.
 
-1. Download the Mac O/S Installer from the [downloads page](https://nodejs.org/en/download/). We recommend the LTS (long-term support) version of Node.js.
-2. Launch the node-v10.x.x.pkg file icon. Click "Continue" for the default options and to install Node.js under the recommended and most common settings. After clicking "Continue" several times, click "Install" to install Node.js. When it is done installing, click "Close".
-3. Verify both Node.js and npm have installed correctly. Open your terminal prompt:
-   - Open your Applications folder
-   - Find and open the Utilities folder
-   - Find "Terminal" and click on it to open it.
-   - Type the following commands in your Terminal
+- **Node.js** >= 10.x
+- **NPM** >= 6.x
 
-Verify Node.js has correctly installed:
+## Databases
 
-```terminal
-node -v
-## You should see "v10.x.x"
-```
-
-Next, verify npm has correctly installed:
-
-```terminal
-npm -v
-## You should see "6.x.x"
-```
-
-:::
-
-::: tab "UBUNTU 18.04"
-
-### Installing Node.js on Ubuntu 18.04
-
-There are multiple methods to install Node.js on _Ubuntu 18.04_.
-
-We will follow the most common download and installation procedure. _These instructions are for Ubuntu 18.04_. (If you are installing on a different version of Ubuntu or a different Linux Distro or if you have trouble following these instructions, please review the [official Node.js documentation](https://nodejs.org/en/docs/).)
-
-1. Install cURL and use cURL to download the Node.js (with npm) source code
-
-```bash
-sudo apt-get install curl
-curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
-```
-
-2. Install Node.js (with npm) and install `build-essential` package:
-
-```bash
-sudo apt-get install -y nodejs
-sudo apt install build-essential
-```
-
-Verify Node.js has correctly installed:
-
-```bash
-node -v
-## You should see "v10.x.x"
-```
-
-Next, verify npm has correctly installed:
-
-```bash
-npm -v
-## You should see "6.x.x"
-```
-
-:::
-::::
+Strapi is supporting multiple databases. If you are using the `--quickstart` or if you manually chose the `SQLite` database, you will not need to install any database on you local computer.
 
-### Using Yarn
+If you want to use a custom database, here is versions you will have to respect:
```

Comment:
We should probably check best LTS versions as I have a feeling this list hasn't been updated in a while. I know right now that MariaDB current LTS is 10.4